spring-integration-zookeeper/gradle.properties

```diff
@@ -0,0 +1 @@
+version=0.0.1.BUILD-SNAPSHOT
```

Comment:
I think I like this versioning manner. And it would be good for our future, when we will have a wish to release it for XD with some existing state and don't wait the whole integration and impl in SI Core.